### ISSUE TITLE
Add useful tooltips on "vision" blocks

### ISSUE THREAD

**Issue by fdila** [CoderBotOrg/backend#138]
Hi!
Currently the tooltip for the vision blocks always displays "Restituisce vero o falso", while the various functions return arrays.
It might be useful to also specify what this arrays contain.

**Comment by avivace** [comment]
Thanks for the report, @fdila !

cc @previ : I (vaguely) recall this already being discussed but eventually we couldn't came up with a simple name (let alone an explanation) for the "array" concept in the UI.

It's also worth mentioning this is also one of the most critical parts for kids to understand how to use (array/how to access an element of an array)

**Comment by previ** [comment]
> Thanks for the report, @fdila !
> 
> cc @previ : I (vaguely) recall this already being discussed but eventually we couldn't came up with a simple name (let alone an explanation) for the "array" concept in the UI.
> 
> It's also worth mentioning this is one of the most critical parts for kids to understand how to use (array/how to access an element of an array)

I agree @avivace, the current implementation is not easy to use. The new block "detect objects" is even more tricky as it returns a structure with multiple data (classes, confidence scores, boundaries).

I did not come up with a usable solution which fits well with blocks.
